### ISSUE TITLE
Installation failed: UnicodeDecodeError

### ISSUE THREAD

**Issue by secsilm** [miyakogi/pyppeteer#38]
Hello, I had a problem when I try to install the package. The following  is my environment.

- Python 3.5.2 :: Anaconda custom (64-bit)
- Windows 10
- pip 9.0.1

Using `pip install pyppeteer` to install the package. The error is 

```
Collecting pyppeteer
  Using cached pyppeteer-0.0.12.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\secsi\AppData\Local\Temp\pip-build-mbsh4qol\pyppeteer\setup.py", line 27, in <module>
        compile_files(in_dir, out_dir, target)
      File "c:\users\secsi\anaconda3\lib\site-packages\py_backwards\compiler.py", line 85, in compile_files
        dependencies.update(_compile_file(paths, target))
      File "c:\users\secsi\anaconda3\lib\site-packages\py_backwards\compiler.py", line 57, in _compile_file
        code = f.read()
    UnicodeDecodeError: 'gbk' codec can't decode byte 0xb6 in position 2147: illegal multibyte sequence

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\secsi\AppData\Local\Temp\pip-build-mbsh4qol\pyppeteer\
```

**Comment by secsilm** [comment]
This error is from https://github.com/kennethreitz/requests-html/issues/84

**Comment by secsilm** [comment]
#https://github.com/kennethreitz/requests-html/issues/84

**Comment by secsilm** [comment]
kennethreitz/requests-html#84

**Comment by secsilm** [comment]
kennethreitz/requests-html#84

**Comment by miyakogi** [comment]
It's `py-backwards`'s error.
Try `pip install py-backwards`.
If it failed, and if possible, please update to python 3.6 and reinstall pyppeteer.
(pyppeteer's python3.5 support is experimental)

**Comment by secsilm** [comment]
@miyakogi Problem solved by updating python to 3.6. Thanks for your suggestion.
